lib/xcode.js

```diff
@@ -171,6 +171,31 @@ async function getCommandLineToolsVersion () {
   return match ? match[1] : undefined;
 }
 
+/**
+ * Check https://trac.macports.org/wiki/XcodeVersionInfo
+ * to see the actual mapping between clang and other components.
+ *
+ * @returns {?string} The actual Clang version in x.x.x.x or x.x.x format,
+ * which is supplied with Command Line Tools. `null` is returned
+ * if CLT are not installed.
+ */
+async function getClangVersion () {
```

Comment:
I take it this will be used in `node-simctl`?

Reply:
I'd like to replace getCommandLineToolsVersion completely because of its instability and `hackiness`. We can remove the function from here after it is replaced in the dependent packages